src/apps/saddle/evmos/saddle.communal-farm.contract-fetcher.ts

```diff
@@ -0,0 +1,65 @@
+import { Inject } from '@nestjs/common';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { SynthetixSingleStakingIsActiveStrategy, SynthetixSingleStakingRoiStrategy } from '~apps/synthetix';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { ContractPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { SaddleCommunalFarm, SaddleContractFactory } from '../contracts';
+import SADDLE_DEFINITION from '../saddle.definition';
+
+const FARMS = [
+  // Saddle Frax 3 Pool Farm
+  {
+    address: '0x0232e0b6df048c8cc4037c52bc90cf943c9c8cc6',
```

Comment:
Pretty sure this is still the wrong address... I'm trying to call `periodFinish()` on it and it doesn't seem to work.

Reply:
If there is no communal farm on Evmos, simply delete this file and remove the balances for this group in the balance fetcher.

Reply:
Removed.